test/screenshot/lib/screenshot.js

```diff
@@ -155,6 +200,32 @@ async function sleep(ms) {
   });
 }
 
+function seconds(seconds) {
```

Comment:
Move these to the top so people like me aren't left wondering where the heck these are defined when I see invocations at the top of the file before these functions

Reply:
Done. I just inlined the functions.